examples/simulation/stable_fluid.py

```diff
@@ -20,8 +20,12 @@
 gravity = True
 debug = False
 paused = False
+sparse_matrix = False
 
-ti.init(arch=ti.gpu)
+if sparse_matrix:
+    ti.init(arch=ti.x64)
+else:
+    ti.init(arch=ti.gpu)
```

Comment:
Instead of hard-coding the `sparse_matrix` argument here, shall we use the `argparse` module and take the user specification from CLI?

You may also want to rename `sparse_matrix` to `use_sparse_matrix`.

Make sure the example still works when launched with no arguments.

Reply:
Sure, thanks for your advice!